### DISCUSSION TITLE
[A-L] (2023/24) Lezione 8 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#179] [Esercizi - Viaggi/Piazza]
Dimostrare la seguente proprietà sull'anello $(\mathbb{Z},+,\cdot)$:
- Siano $a,b,c \in \mathbb{Z},\ a\neq 0,\ (b,c)\neq (0,0)$,  $MCD(ab,ac)=|a|MCD(b,c)$

**Comment by Elia-Belli** [comment]
Sia $d=MCD(ab,ac)=|a|MCD(b,c)$ e $d'= MCD(b.c)$
per **(Bezout)** : $d=(ab)x_0+(ac)y_0$, e $d'=bx_1+cy_1$
$d=|a|d' \Rightarrow (ab)x_0+(ac)y_0 = |a|(bx_1+cy_1) \Rightarrow a(bx_0+cy_0) = |a|(bx_1+cy_1)$
Questa uguaglianza è verificata per $(x_1=x_0,y_1=y_0)$ se $a>0$, e per $(x_1=-x_0,y_1=-y_0)$ se $a<0$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/71b464f0-794c-4084-a8f7-69cf0b479090)
